src/btree/bt_debug.c

```diff
@@ -1450,7 +1489,10 @@ __debug_row_skip(WT_DBG *ds, WT_INSERT_HEAD *head)
     session = ds->session;
 
     WT_SKIP_FOREACH (ins, head) {
-        WT_RET(__debug_item_key(ds, "insert", WT_INSERT_KEY(ins), WT_INSERT_KEY_SIZE(ins)));
+        if (F_ISSET(ds, WT_DEBUG_DUMP_APP_DATA))
```

Comment:
Could these be rolled up into `debug_item_key` and `debug_item_value` instead of replacing their callers?